api/IDBCursor.json

```diff
@@ -104,7 +104,7 @@
           "spec_url": "https://w3c.github.io/IndexedDB/#ref-for-dom-idbcursor-advance①",
           "support": {
             "chrome": {
-              "version_added": "23"
+              "version_added": "24"
```

Comment:
The parent feature says this was prefixed in 23. The collector tests aren't enough to detect this, testing only the unprefixed prototype, so I think most of these updates are wrong.